### PR TITLE
zimg: update to 2.1.

### PR DESCRIPTION
**Pull request by lemmi**

### CODE REVIEW

**Comment by lemmi** [comment]
I filed an issue upstream.

**Comment by ebfe** [comment]
https://github.com/ebfe/void-packages/commit/170b02932c51bbca5530bdf67de101290de58caf

**Comment by Gottox** [comment]
Any news on this?

**Comment by Vaelatern** [comment]
@lemmi please see the patch @ebfe provided, and possibly mention it to upstream.

**Comment by lemmi** [comment]
there isn't a release yet with the fix. the fix is from after version 2.1, which fails to build unfortunately. if anyone instists on this update (vapoursynth beeing the only package using this afaik), i'll patch it, but i don't really see the point.

**Comment by Vaelatern** [comment]
@lemmi Please include the patch as in @ebfe's branch in this PR. Just put a comment in the template file linking to the issue where you reported to upstream that something was wrong.

File: srcpkgs/zimg/template [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Template file for 'zimg'
 pkgname=zimg
-version=2.0.4
+# 2.1 needs a patch to cross compile.
+# Please remove un update
```
Comment (Vaelatern) [inline srcpkgs/zimg/template]: typo ;)

**Comment by Vaelatern** [comment]
Seems like the shasum changed on you.

**Pull request merged**